### PR TITLE
add sso loing docs

### PR DESCRIPTION
**Pull request by anwen-anyi**
add sso loing docs

### CODE REVIEW

**Comment by Xhofe** [comment]
Did you test all right? It seems here there are bugs
https://github.com/alist-org/alist-web/pull/61#issuecomment-1447962254

**Comment by anwen-anyi** [comment]
呃呃呃...原来GitHub改动过 GitHub现在似乎有问题还得改改

File: docs/guide/advanced/sso.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+tag:
+  - Advanced
+  - Guide
+# this page is sticky in article list
+sticky: true
+# this page will appear in starred articles
+star: true
+---
+
+# Third-party SingleSignOn
```
Comment (itsHenry35) [inline docs/guide/advanced/sso.md:19]: Third-party single sign-on, single sign-on shall be separated

Comment (anwen-anyi) [reply docs/guide/advanced/sso.md:19]: Third-party single sign-on 这样是吗？

Comment (itsHenry35) [reply docs/guide/advanced/sso.md:19]: > Third-party single sign-on 这样是吗？

对的

File: docs/guide/advanced/sso.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+### Github client id
+
+client ID
+
+### Github client secrets
+
+secrets ID
+
+
```
Comment (itsHenry35) [inline docs/guide/advanced/sso.md:28]: 不理解这几行放这有啥用

**Pull request merged**